services/plans/plan_states/main.go

```diff
@@ -1,10 +1,10 @@
-package plan_states
+package new_plan_states
```

Comment:
@coutinol Why can't we rename the plan-states to "pass-plan-states" so that the recent plan-states are named "plan-states" ?

Reply:
hmm I guess we can, but in that case we need to ensure that people using pass plan states migrate their scripts when bumping the leanspace provider. Then, as it it is a breaking change, we need to bump the major digit on the delivered terraform provider package. I would then suggest to talk to @jerome-leanspace to find out who is using them

Reply:
[According to our last check](https://leanspace.atlassian.net/wiki/spaces/ET/pages/2528346140/Solutions+-+Deprecation+to+handle) only Infinite and Clearspace are using Pass Plans
So we should be on the safe side, therefore we have 2 options:
- Just remove the pass plan code and keep only the new one (Ideally we put somewhere in the terraform docs that the latest version working with pass plan is version x.y.z)
- Keep both I would still prefer to have another name for the plan states